### ISSUE TITLE
Can't install on M1

### ISSUE THREAD

**Issue by Zettt** [jotta/jotta-cli-issues#155]
Hello, 

I can't install jotta-cli on my M1 Mac. The brew command gives this error:

```
brew install jotta/cli/jotta-cli
==> Installing jotta-cli from jotta/cli
jotta-cli: The intel architecture is required for this software.
```

It doesn't even install using Rosetta 2. 

Just hope you will be able to provide an M1 build in the future. Thank you.

**Comment by coopersimon** [comment]
Hi Zettt,

This is on our roadmap. I'll let you know when it's available.

**Comment by coopersimon** [comment]
Hi again,

The newest version we just released (0.11.47628) supports Apple Silicon.
